pinot-common/src/main/java/org/apache/pinot/common/segment/generation/SegmentGenerationUtils.java

```diff
@@ -229,4 +233,63 @@ private static String fetchUrl(URL url, String authToken)
     }
     return IOUtils.toString(connection.getInputStream(), StandardCharsets.UTF_8);
   }
+
+
+  /**
+   * Find matching files from root directory specified in fileUri.
+   * If includePattern and excludePattern are not null, get all the files that match includePattern and exclude files
+   * that match excludePattern.
+   * If
+   *
+   * @param pinotFs root directly fs
```

Comment:
(minor) You mean `directory` here? Same for `fileUri`

Reply:
Yeah. Typo